webapp/channels/src/components/admin_console/access_control/modals/user_sync/user_sync_modal.scss

```diff
@@ -0,0 +1,60 @@
+.user-list-modal-dialog {
+    width: 660px;
+    max-width: 90vw;
+    padding: 0;
+
+    .modal-body {
+        max-height: fit-content;
+        padding: 0;
+    }
+
+    .col-sm-12 {
+        padding-top: 12px;
+        padding-bottom: 12px;
+        padding-left: 0;
+    }
+
+    .col-xs-12 {
+        padding-right: 0;
+        padding-left: 0;
+    }
+
+    .more-modal__row {
+        padding: 10px 0;
+    }
+}
+
+.tabs {
+    display: flex;
+    padding: 0 16px;
+    border-bottom: 1px solid var(--center-channel-color-08);
+}
+
+.tab-button {
+    padding: 10px 15px;
+    border: none;
+    margin-bottom: -1px;
+    background: none;
+    color: var(--center-channel-color-64);
+    cursor: pointer;
+    font-weight: 600;
+
+    &.active {
+        border-bottom: 2px solid var(--button-bg);
+        color: var(--button-bg);
+    }
+
+    &:hover:not(.active) {
+        color: var(--center-channel-color);
+    }
+}
+
+.tab-content {
```

Comment:
This class name is definitely shared with some Bootstrap classes that are used throughout the app, and I think that might also apply to `tabs` and `tab-button` above. Could you move these into a class for just this component to prevent them from affecting other parts of the app?